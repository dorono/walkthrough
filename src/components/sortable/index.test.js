import React from 'react';
import {shallow} from 'enzyme';
import Sortable from './index';

const defaultProps = {
    children: () => null,
    items: [],
    sortOptions: [{label: 'label', func: jest.fn}],
};

describe('Sortable', () => {
    it('should render without errors', () => {
        shallow(<Sortable {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<Sortable {...defaultProps} />)).toMatchSnapshot();
    });
});
