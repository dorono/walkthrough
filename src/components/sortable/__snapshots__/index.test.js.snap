// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sortable should match snapshot 1`] = `""`;
